.xo-config.json

```diff
@@ -8,25 +8,28 @@
 		"__filebasename"
 	],
 	"rules": {
+		"no-alert": "off",
+		"no-warning-comments": "off",
+		"unicorn/expiring-todo-comments": ["warn", {
+			"allowWarningComments": false
+		}],
 		"@typescript-eslint/naming-convention": "off",
 		"@typescript-eslint/no-unsafe-assignment": "off",
 		"@typescript-eslint/no-unsafe-member-access": "off",
 		"@typescript-eslint/no-unsafe-return": "off",
 		"@typescript-eslint/no-unsafe-call": "off",
-
-		"import/no-cycle": "off",
-		"import/no-unassigned-import": "off",
-		"import/extensions": "off",
-		"no-alert": "off",
-		"react/jsx-key": "off",
-		"node/prefer-global/process": "off",
 		"@typescript-eslint/consistent-type-definitions": "error",
 		"@typescript-eslint/explicit-function-return-type": [
 			"error",
 			{
 				"allowExpressions": true
 			}
 		],
+		"node/prefer-global/process": "off",
+		"import/no-cycle": "off",
```

Comment:
I think this is disabled in the shared config already

Reply:
Do you mean [xo-react](https://github.com/xojs/eslint-config-xo-react/blob/main/index.js)? It doesn't seem so.

Reply:
XO itself: https://github.com/xojs/xo/blob/7da6311abcd069bea561a9be0f9b7aa220784a34/config/plugins.cjs#L215-L220

But actually it is enabled there, so ignore the comment